benchmarks/src/generated/main/grpc/io/grpc/benchmarks/proto/BenchmarkServiceGrpc.java

```diff
@@ -271,14 +271,23 @@ public void invoke(Req request, io.grpc.stub.StreamObserver<Resp> responseObserv
 
   private static io.grpc.ServiceDescriptor serviceDescriptor;
 
-  public static synchronized io.grpc.ServiceDescriptor getServiceDescriptor() {
-    if (serviceDescriptor == null) {
-      serviceDescriptor = new io.grpc.ServiceDescriptor(SERVICE_NAME,
-          new BenchmarkServiceDescriptorSupplier(),
-          METHOD_UNARY_CALL,
-          METHOD_STREAMING_CALL);
+  public static io.grpc.ServiceDescriptor getServiceDescriptor() {
+    if (serviceDescriptor != null) {
```

Comment:
concurrent calls can not get two copies because `newServiceDescriptor()` is synchronized, the serviceDescriptor field can not be overwritten, only the same instance can be returned.